teamcity: failed test: TestDockerCLI

The following tests appear to have failed on master (acceptance): TestDockerCLI/test_exec_log.tcl, TestDockerCLI

You may want to check [for open issues](https://github.com/cockroachdb/cockroach/issues?q=is%3Aissue+is%3Aopen+TestDockerCLI).

[#1144311](https://teamcity.cockroachdb.com/viewLog.html?buildId=1144311):


```
TestDockerCLI/test_exec_log.tcl
--- FAIL: acceptance/TestDockerCLI: TestDockerCLI/test_exec_log.tcl (3.950s)




TestDockerCLI
--- FAIL: acceptance/TestDockerCLI (264.940s)

------- Stdout: -------
    test_log_scope.go:81: test logs captured to: /home/agent/work/.go/src/github.com/cockroachdb/cockroach/artifacts/acceptance/logTestDockerCLI067056527
    test_log_scope.go:62: use -show-logs to present logs inline




```

Please assign, take a look and update the issue accordingly.

--- a/pkg/cli/interactive_tests/test_exec_log.tcl
+++ b/pkg/cli/interactive_tests/test_exec_log.tcl
@@ -42,8 +42,18 @@ eexpect root@
 flush_server_logs
 
 # Now check the items are there in the log file.
-system "grep -q 'hello.*world' $logfile"
-system "grep -q nonexistent $logfile"
+# We need to iterate because flush_server_logs
+# only syncs on flush of cockroach.log, not
+# the exec log.
+system "for i in `seq 1 3`; do
+  grep 'hello.*world' $logfile &&
+  grep nonexistent $logfile &&
+  exit 0;
+  echo still waiting;
+  sleep 1;
+done;
+echo 'server failed to flush exec log?'
+exit 1;"
 system "if grep -q lovely $logfile; then false; fi"
 
 end_test

